osf/management/commands/sync_doi_metadata.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from osf.models import Identifier
+from django.utils import timezone
+
+from framework.celery_tasks import app
+from framework.celery_tasks.handlers import enqueue_task
+
+logger = logging.getLogger(__name__)
+
+
+@app.task(name='osf.management.commands.sync_doi_metadata', max_retries=5, default_retry_delay=60)
+def sync_identifier_doi(identifier):
+    identifier.referent.request_identifier_update('doi')
+    identifier.modified = timezone.now()
```

Comment:
Minor: `.save` will update this as it's `auto_now=True`; there's no real need to set it.